core/common/src/ClipStyle.ts

```diff
@@ -115,6 +172,8 @@ export interface ClipStyleProps {
   insideColor?: RgbColorProps;
   /** If defined, geometry outside of the clip planes will be drawn in this color instead of being clipped. */
   outsideColor?: RgbColorProps;
+  /** If defined, pixels inside the clip planes within a width will be highlighted */
+  clipHighlight?: ClipHighlightProps;
```

Comment:
```suggestion
  highlight?: ClipHighlightProps;
```

We already know it applies to the clip by virtue of being a field of `ClipStyle`.